lib/go-tc/crstates.go

```diff
@@ -37,10 +38,12 @@ type CRStatesDeliveryService struct {
 
 // IsAvailable contains whether the given cache or delivery service is available. It is designed for JSON serialization, namely in the Traffic Monitor 1.0 API.
 type IsAvailable struct {
-	IsAvailable    bool `json:"isAvailable"`
-	Ipv4Available  bool `json:"ipv4Available"`
-	Ipv6Available  bool `json:"ipv6Available"`
-	DirectlyPolled bool `json:"-"`
+	IsAvailable    bool      `json:"isAvailable"`
+	Ipv4Available  bool      `json:"ipv4Available"`
+	Ipv6Available  bool      `json:"ipv6Available"`
+	DirectlyPolled bool      `json:"-"`
+	Status         string    `json:"status"`
+	LastHmPoll     time.Time `json:"lastHmPoll"`
```

Comment:
Can we just call this `LastPoll`? The `Hm` bit is kind of unnecessary/already understood.